elastic-utils-lib/src/cmr/elastic_utils/es_index_helper.clj

```diff
@@ -60,6 +60,16 @@
              {:content-type :json
               :body {:actions actions}}))
 
+;; We have to roll our own get-aliases function because Elasticsearch route on GET alias
+;; for an index has changed and clojurewerkz is outdated
+(defn get-aliases
+  "get index aliases"
+  [conn index-name]
+  (let [aliases-url (rest/url-with-path conn index-name "_alias")
```

Comment:
this value, _alias, is mentioned in more then one file, should it be a constant in case elastic decides to change it again?